lib/internal/streams/writable.js

```diff
@@ -433,14 +433,22 @@ function Writable(options) {
   }
 }
 
+function hasInstanceFn(instance) {
+  if (FunctionPrototypeSymbolHasInstance(this, instance)) return true;
+  if (this !== Writable) return false;
+
+  return instance && instance._writableState instanceof WritableState;
+};
+ObjectDefineProperty(hasInstanceFn, 'name', {
+  __proto__: null,
+  value: '[Symbol.hasInstance]',
+  writable: false,
+  enumerable: false,
+  configurable: true,
+});
 ObjectDefineProperty(Writable, SymbolHasInstance, {
   __proto__: null,
```

Comment:
Let's set up a util to simplify that
```suggestion
ObjectDefineProperty(Writable, SymbolHasInstance, {
  __proto__: null,
  value: setName(function(object) {
    if (FunctionPrototypeSymbolHasInstance(this, object)) return true;
    if (this !== Writable) return false;

    return object && object._writableState instanceof WritableState;
  }, '[Symbol.hasInstance]'),
```

Reply:
Yep. I'm thinking of util that would take the symbol directly and `[${kPrivateProperty.toString()}]`-fy it.
Fixing something like
```js
for (const prop of Object.getOwnPropertySymbols(require('console').Console.prototype))
  console.log(prop, '-->', require('console').Console.prototype[prop]);
```
will definitely require a helper function. 😅